### PR TITLE
Redirects leaderboards blog

### PR DESCRIPTION
**Pull request by clefourrier**
We now have a homogeneous naming pattern:
- Open LLM Leaderboard related: ollm-leaderboard-topic
- Partner leaderboard: leaderboard-partner or leaderboard-topic
- Partner arena: arena-partner or arena-topic

First use of #1883

### CODE REVIEW

**Comment by julien-c** [comment]
`ollm` is maybe a bit weird? `llm-leaderboard` was maybe clearer? (up to you ofc)

File: _redirects.yml [hunk 1]
```diff
@@ -1,4 +1,18 @@
...
+leaderboards-on-the-hub-upstage: leaderboard-upstage
+leaderboards-on-the-hub-haizelab: leaderboard-haizelab
+leaderboards-on-the-hub-patronus: leaderboard-patronus
+leaderboards-on-the-hub-nphardeval: leaderboard-nphardeval
+leaderboards-on-the-hub-decodingtrust: leaderboard-decodingtrust
+leaderboards-on-the-hub-hallucinations: leaderboard-hallucinations
+llm-leaderboard: ollm-leaderboard-rlhf
+evaluating-mmlu-leaderboard: ollm-leaderboard-mmlu
+leaderboard-drop-dive: ollm-leaderboard-drop
+zh/leaderboards-on-the-hub-patronus.md : zh/leaderboard-patronus.md
```
Comment (mishig25) [inline _redirects.yml:15]: for the chinese blog, create `_redirects.yml` inside `zh` folder & put the values without `zh` prefix

Comment (clefourrier) [reply _redirects.yml:15]: Ok will do thank you

**Comment by clefourrier** [comment]
Hm I have no strong opinion, wdyt of `hf-llm-leaderboard-topic`  or `open-llm-leaderboard-topic` ? I assume we'll get lots of other cool llm leaderboards in the future :D

**Comment by mishig25** [comment]
Note: should be merged after the next deploy of hf.co as the changes are not live yet

**Comment by julien-c** [comment]
i like `open-llm-leaderboard-xxx` but up to you!

**Review by mishig25 — APPROVED** [review]
LGTM !

**Pull request merged**